pkg/nfs/driver.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nfs
+
+import (
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	"github.com/golang/glog"
+
+	csicommon "github.com/kubernetes-csi/drivers/pkg/csi-common"
```

Comment:
I believe "github.com/kubernetes-csi/drivers" is supposed to be deprecated once all the drivers have been moved to their own respective repo. Maybe we should not use this library?

Reply:
@mathu97 right , but i can see there is `https://github.com/kubernetes-csi/csi-lib-common` where we will move out the csi-common code base , which can be used later by the other drivers.

Reply:
Oh okay, i didn't realize that there was such a repo. Sounds good.